### PR TITLE
DW-19: Add styles to Menu section

### PR DESCRIPTION
**Pull request by TomiJortiz**
# [DW-19: Add styles to Menu section](https://trello.com/c/uI9sLRM1/122-dw-19-add-styles-to-menu-section)

> Ticket code, title of the ticket and as a hyperlink, the corresponding TRELLO link

## Type of change

> Exchange rate entered. The possible values ​​for this section are:
>
> - Feature (new feature of the server/application)

## Description

> - Add styles to Menu section

## Attachments
image

image

image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
     "cSpell.words": [
         "Restaurantly",
         "stylelint"
-    ]
+    ],
+    "liveServer.settings.port": 5501
```
Comment (maxidelfino) [inline .vscode/settings.json:6]: Why this code line?

File: src/pages/menu-specials/index.css [hunk 2]
```diff
@@ -1,7 +1,130 @@
-.title {
+/* stylelint-disable no-descending-specificity */
```
Comment (maxidelfino) [inline src/pages/menu-specials/index.css:1]: Delete this comment

File: src/pages/menu-specials/index.css [hunk 3]
```diff
@@ -1,7 +1,130 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;600;700;800&family=Poppins:wght@200;600&family=Taviraj:wght@500&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
```
Comment (maxidelfino) [inline src/pages/menu-specials/index.css:9]: Remove this font-family

File: src/pages/menu-specials/index.css [hunk 4]
```diff
@@ -1,7 +1,130 @@
...
+.menu-container__title > h3 {
+  font-family: 'Playfair Display', sans-serif;
+  font-size: 45px;
+  font-weight: 600;
+  color: #cda45e;
+  text-align: center;
+}
+
+.father-container {
+  /* width: 1000px; */
```
Comment (maxidelfino) [inline src/pages/menu-specials/index.css:63]: Delete this comment

File: src/pages/menu-specials/index.css [hunk 5]
```diff
@@ -1,7 +1,130 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;600;700;800&family=Poppins:wght@200;600&family=Taviraj:wght@500&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+.menu-container {
```
Comment (maxidelfino) [inline src/pages/menu-specials/index.css:12]: This code is repeated in several parts:

```
{
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

We should avoid repeating code. Maybe we could do the following:

```
.menu-container,
.menu-container__title,
.father-container__item {
  display: flex;
  flex-direction: column;
  align-items: center;
}
```

```
.menu-container {
  padding: 0;
  background-color: #1a1a1a;
}

.menu-container__title {
  justify-content: center;
  color: #ffffff;
}

.father-container__item {
  width: 20%;
  min-width: 380px;
  margin-top: 30px;
  margin-bottom: 50px;
}
```

File: src/pages/menu-specials/index.html [hunk 6]
```diff
@@ -55,84 +55,194 @@
         </div>
       </nav>
     </header>
+    <!-- bootstrap trabaja asi: .container - .container-fluid > .row > col-*-* -->
```
Comment (maxidelfino) [inline src/pages/menu-specials/index.html:58]: Delete this comment

File: src/pages/menu-specials/index.html [hunk 7]
```diff
@@ -195,3 +305,4 @@ <h3>Every day: 11 AM - 23 PM</h3>
       </footer>
   </body>
 </html>
+me
```
Comment (maxidelfino) [inline src/pages/menu-specials/index.html:308]: Remove this

File: src/pages/menu-specials/index.css [hunk 8]
```diff
@@ -1,7 +1,130 @@
...
+.father-container__content > div {
+  display: flex;
+  align-items: flex-start;
+}
+
+.father-container__content > div > a {
+  font-size: 16px;
+  font-weight: 800;
+  color: #ffffff;
+  text-decoration: none !important;
```
Comment (maxidelfino) [inline src/pages/menu-specials/index.css:95]: Delete this `important`

**Review by maxidelfino — CHANGES_REQUESTED** [review]

**Review by LautaroTinelli — APPROVED** [review]

**Review by LautaroTinelli — APPROVED** [review]

**Review by maxidelfino — APPROVED** [review]
🚀

**Pull request merged**